File: 0-9/8.go
package main

import "fmt"

// 题目: 跳台阶

/**
 *
 * @param number int整型
 * @return int整型
 */
func jumpFloor(number int) int {
	// write code here
	if number == 0 {
		return 0
	}
	numOne := 1
	numTwo := 2
	for i := 2; i <= number; i++ {
		numOne, numTwo = numTwo, numOne+numTwo
	}

	return numOne
}

func main() {
	fmt.Println(jumpFloor(4))
}

File: 0-9/8.py
# -*- coding:utf-8 -*-
# 题目: 跳台阶

class Solution:
    def jumpFloor(self, number):
        # write code here
        firstNum = 1
        secondNum = 2
        index = 2
        while index <= number:
            firstNum, secondNum = secondNum, firstNum + secondNum
            index += 1
        return firstNum


obj = Solution()
print(obj.jumpFloor(6))
